src/fsdgenaspnet/fsdgenaspnet.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net6.0</TargetFrameworks>
```

Comment:
Out of curiosity, if one has only a .NET 5 runtime installed, would this run the 3.1 version?

Reply:
I had the same thought.